test/SdkE2ETests/xunit.runner.json

```diff
@@ -0,0 +1,5 @@
+{
+  "$schema": "https://xunit.net/schema/current/xunit.runner.schema.json",
+  "parallelizeAssembly": false,
+  "parallelizeTestCollections": false
```

Comment:
I noticed that locally there were multiple tests trying to run simultaneously, which would explain some of the weird errors we were getting. Tests were trying to build/rebuild the same project at the same time. I think this will fix it.